frontend/providers/applaunchpad/public/locales/zh/translation.json

```diff
@@ -0,0 +1,123 @@
+  "Advanced Configuration": "高级配置",
+  "Start Command": "启动命令",
+  "Run Parameters": "运行参数",
+  "Not Configured": "未配置",
+  "Accessible to the Public": "可外网访问",
+  "Configuration File": "配置文件",
+  "Storage Volume": "存储卷",
+  "Environment Variables": "环境变量",
+  "Fixed Number of Instances": "固定实例数",
+  "Image Name": "镜像名",
+  "Private": "私有",
+  "Public": "公开",
+  "Image Name (Private)": "镜像名（私有）",
+  "Real-time Monitoring": "实时监控",
+  "Network Configuration": "网络配置",
+  "Intranet Address": "内网地址",
+  "Internet Address": "外网地址",
+  "Not Enabled": "未开启",
+  "Update Time": "更新时间",
+  "Continue": "继续",
+  "Terminal": "终端",
+  "Log": "日志",
+  "None": "暂无",
+  "Intelligent Analysis": "智能分析",
+  "Problem Analysis": "问题分析",
+  "Export": "导出",
+  "Edit Environment Variables": "编辑环境变量",
+  "Application Deployment": "应用部署",
+  "Creating": "创建中",
+  "Abnormality Detected": "有异常",
+  "Change Application": "变更应用",
+  "Deploy Application": "部署应用",
+  "Confirm Deploy Application?": "确认部署应用?",
+  "Deployment Successful": "部署成功",
+  "Deployment Failed": "部署失败",
+  "Update Application": "更新应用",
+  "Confirm Update Application?": "确认更新应用?",
+  "Update Successful": "更新成功",
+  "Update Failed": "更新失败",
+  "Prompt": "提示",
+  "jump_prompt": "跳转提示",
+  "jump_message": "该应用不允许单独使用，点击确认前往 Sealos Desktop 使用。",
+  "Please note that application cannot be modified while in paused state, and if you are using storage volumes, storage charges will still apply. Please confirm!": "请注意，暂停状态下无法变更应用，并且如果您使用了存储卷，存储券仍会收费，请确认！",
+  "Confirm to restart this application?": "确认重启该应用？",
+  "You haven't created any application yet.": "您还没有新建应用。",
+  "Confirm deletion": "确认删除",
+  "Deletion warning": "删除警告",
+  "Are you sure you want to delete this application? If you proceed, all data for this project will be deleted.": "如果确认要删除这个应用吗？如果执行此操作，将删除该项目的所有数据。",
+  "Please enter": "请输入",
+  "Elastic scaling": "弹性伸缩",
+  "Fixed instance": "固定实例",
+  "Stateful": "有状态",
+  "Stateless": "无状态",
+  "Image name cannot be empty.": "镜像名不能为空。",
+  "Please confirm to restart the Pod?": "请确认重启 Pod ?",
+  "Limit CPU": "Limit CPU",
+  "Limit Memory": "Limit Memory",
+  "CPU target value": "cpu目标值",
+  "Memory target value": "内存目标值",
+  "Reboot Success": "重启成功",
+  "success": "成功",
+  "target_value": "目标值",
+  "Edit Env Variable": "编辑环境变量",
+  "public": "公共",
+  "private": "私有",
+  "Config Form": "配置表单",
+  "file": "文件",
+  "Mirror Source": "镜像源",
+  "App Name": "应用名称",
+  "Basic Config": "基础配置",
+  "YAML File": "YAML 文件",
+  "filename": "文件名",
+  "file value": "文件值",
+  "local storage": "本地存储",
+  "multiple instances do not share data": "多个实例间数据不互通",
+  "capacity": "容量",
+  "mount path": "挂载路径",
+  "Expose Container Ports": "容器暴露端口",
+  "CPU target is the CPU utilization rate of any container": "CPU 目标值为任一容器的 CPU 利用率",
+  "protocol": "协议",
+  "exportDomain": "出口域名",
+  "Custom domain": "自定义域名",
+  "Please CNAME your custom domain to": "请将您的自定义域名 cname 到",
+  "Option": "选填",
+  "Run command": "运行命令",
```

Comment:
duplicated

Reply:
The Chinese commands corresponding to Command are different, one is the start command, and the other is the run command, which should be uniformly changed to Command